Neos.Flow/Classes/Error/AbstractExceptionHandler.php

```diff
@@ -240,15 +240,16 @@ protected function resolveRenderingGroup($exception)
                 }
             }
         }
+        return null;
     }
 
     /**
      * Formats and echoes the exception and its previous exceptions (if any) for the command line
      *
-     * @param object $exception \Exception or \Throwable
+     * @param \Throwable $exception
      * @return void
      */
-    protected function echoExceptionCli($exception)
+    protected function echoExceptionCli(\Throwable $exception)
```

Comment:
..but you _did_ add it here?!

Reply:
I think that was just because it is implemented here, not api and hence my thinking process was probably: "No one overrides it anyway". But you're right that's inconsequent. It probably needs adjustment by people that implement an ExceptionHandler anyway, so we can just go full out with typehints and leave a nice message about this, or maybe even provide a migration?